Automation/update_illustrations.py

```diff
@@ -43,6 +43,7 @@
 '''
 
 illustration_source_template = '''import Foundation
+import Orbit
```

Comment:
Improvement suggestion: Any idea for naming to improve the call site? We currently have it appear as an `Illustration."Image"` type that looks confusing ( = same name as native `Image` view type).

Maybe: `IllustrationImage` type? (We have `Symbol` for `Icon`)
 
<img width="565" alt="image" src="https://github.com/kiwicom/orbit-swiftui/assets/35844477/d5ad21ed-b104-4de9-b63f-1a47d3dc52b7">

Reply:
You mean we would have `Illustration.IllustrationImage` or no nesting and just `IllustrationImage` at top level?

Reply:
Could be either. But in the nesting variant, it should be something like `Icon.Symbol` (`Illustration.Symbol`  would work for consistency, even if it may sound strange) that does not clash with a known SwiftUI name type.

Reply:
Another idea is `Illustration.Asset` (hinting about actual `xcasset` content that Illustrations use)

Reply:
Used `Illustration.Asset`